lib/internal/bootstrap/node.js

```diff
@@ -319,6 +319,18 @@ ObjectDefineProperty(process, 'features', {
 }
 
 const { emitWarning, emitWarningSync } = require('internal/process/warning');
+const { getOptionValue } = require('internal/options');
+
+let hasTypeScriptSupport;
+ObjectDefineProperty(process.features, 'typescript', {
```

Comment:
I believe the name of `typescript` would be ambiguous about which flag is enabled when https://github.com/nodejs/node/pull/54283 lands, `--experimental-strip-types` or `--experimental-transform-types`.